trakt_scrobbler/trakt_interface.py

```diff
@@ -40,6 +40,12 @@ def get_trakt_id(title, item_type, year=None):
 
     logger.debug(f'Searching trakt: Title: "{title}"{year and f", Year: {year}" or ""}')
     results = search(title, [required_type], year)
+    if (results == [] and year is not None): # no match, possibly a mismatch in year metadata
```

Comment:
Parentheses not needed, line length below 88.
```suggestion
    if results == [] and year is not None:
    # no match, possibly a mismatch in year metadata
```